### PR TITLE
🗃 Add connection to database and getCollection function

### LINKED ISSUE #5: MongoDB
- [x] Set-up MongoDB

- [x] Add Test-Data-Structure into MongoDB

### LINKED ISSUE #6: Set-up express.JS
- [x] Add functions to access data from database

- [x] GET-function

- [x] POST-function

- [x] PATCH-function

- [x] DELETE-function
- [ ] Hotel for nicer dev urls with https

### PR DESCRIPTION
**Pull request by arassaria**
# Description

Lot of cool stuff depending on the mongoDB coming in this pull request. 😄 🚀 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by bwnstck — APPROVED** [review]
[lgtm🚀](https://media.giphy.com/media/CDZwopbecAbIc/giphy.gif)

File: lib/serverMethods.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const { collection } = require("./database");
+
+async function getCollection(collectionName) {
+  const promises = await collection(collectionName).find({}).toArray();
+  const data = await promises;
+  return data;
```
Comment (lmachens) [inline lib/serverMethods.js]: I think `promises` is already your `data`.

```suggestion
  return await collection(collectionName).find({}).toArray();
```

File: db.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
   "plannedTasks": [
```
Comment (lmachens) [inline db.json]: This file could be removed in this pull request if it is not used.
In addition, remove `json-server` from your project.

**Review by lmachens — COMMENTED** [review]
well played 🎯

File: lib/serverMethods.js (hunk 1 above)
Comment (arassaria) [reply lib/serverMethods.js]: Thanks for the feedback. Added the change into the project 🚀

**Review by bwnstck — APPROVED** [review]
lets go🚀

**Pull request merged**